Dockerfile

```diff
@@ -1,39 +1,55 @@
-ARG base_image=ruby:2.7.6
-FROM ${base_image}
-
-# Add yarn to apt sources
-RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
-RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
-
-RUN apt-get update -qq && apt-get upgrade -y
-RUN apt-get install -y build-essential nodejs yarn && apt-get clean
-
-# This image is only intended to be able to run this app in a production RAILS_ENV
-ENV RAILS_ENV production
-
-ENV APP_HOME /app
-ENV DATABASE_URL mysql2://root:root@mysql/signon
-ENV GOVUK_APP_NAME signon
-ENV PORT 3016
-ENV TEST_DATABASE_URL mysql2://root:root@mysql/signon_test
-ENV ASSETS_PREFIX /assets/signon
-
-RUN mkdir $APP_HOME
-
-WORKDIR $APP_HOME
-ADD Gemfile* .ruby-version $APP_HOME/
-RUN bundle config set deployment 'true'
-RUN bundle config set without 'development test'
-RUN bundle install --jobs 4
-
-ADD . $APP_HOME
-
-RUN GOVUK_APP_DOMAIN=www.gov.uk \
-  DEVISE_PEPPER=`openssl rand -base64 40` \
-  DEVISE_SECRET_KEY=`openssl rand -base64 40` \
-  GOVUK_WEBSITE_ROOT=https://www.gov.uk \
-  bundle exec rails assets:clean assets:precompile
-
-HEALTHCHECK CMD curl --silent --fail localhost:$PORT || exit 1
-
+ARG base_image=ruby:2.7.6-slim-bullseye
+
+FROM $base_image AS builder
+ENV RAILS_ENV=production \
+    NODE_ENV=production \
+    GOVUK_APP_DOMAIN=www.gov.uk \
+    GOVUK_WEBSITE_ROOT=https://www.gov.uk \
```

Comment:
Should we change these to be "unused" as per the DEVISE examples since we don't expect them to actually affect the app (since it'll probably cause integration/staging problems if they do). 

It's pretty annoying they're needed at boot time rather than at runtime given they're only for plek.

Reply:
Thanks!